reference-data-proxy/src/v1/controllers/currency-exchange-rate.controller.ts

```diff
@@ -20,10 +61,14 @@ export const getExchangeRate = async (req: Request, res: Response) => {
     actualTarget = source;
   }
 
+  const url = exchangeRateDate
```

Comment:
80% of this is the same on either side. I wonder if this is easier to read:
```js
  let url = `${exchangeRateURL}currencies/exchange?source=${actualSource}&target=${actualTarget}`;
  if (exchangeRateDate) {
    url = `${url}&exchangeRateDate=${exchangeRateDate}`
  }
```

Reply:
Have interpolated above.